### ISSUE TITLE
The translation is alive?

### ISSUE THREAD

**Issue by SergioBG** [jhenahan/ohaskell-translations#14]
I'm Sorry but the proyect is alive?? you are continue with the translation of book?

**Comment by denisshevchenko** [comment]
Of course, project is alive! It's just a little pause... In fact, I wrote a new chapters about monad transformers, so first volume of book nears to end. So in the nearest future translation will continue.

**Comment by SergioBG** [comment]
I like this ;) , thanks for your work with this book.

**Comment by jhenahan** [comment]
Yeah, sorry, I just started a new job so I've been a little busy. The project is very much still alive!

**Comment by denisshevchenko** [comment]
I finally figured out the transformers ReaderT, WriterT and StateT. So beautiful and so useful tools! Unfortunately, many novices think that transformers is something like sacral mystery. I believe my explanations will destroy such opinion...

**Comment by zudov** [comment]
I've also been a little busy, but now I gonna contribute a little more. As @denisshevchenko did something the chapter on monad transformers I'll pick something else.
